misk-slack/src/main/kotlin/slack-api/RealSlackClient.kt

```diff
@@ -0,0 +1,27 @@
+package `slack-api`
+
+import retrofit2.Response
+import java.io.IOException
+import java.io.UncheckedIOException
+import javax.inject.Inject
+
+class RealSlackClient @Inject constructor(
+  private val slackApi: SlackApi
+) : SlackClient {
+  override fun postMessage(request: PostMessageJson) : PostMessageResponseJson {
+    return callSlack{slackApi.postMessage(request).execute()}
+  }
+  override fun postConfirmation(url: String, request: PostMessageJson) : PostMessageResponseJson {
+    return callSlack{slackApi.postConfirmation(url, request).execute()}
```

Comment:
```suggestion
    return callSlack{ slackApi.postConfirmation(url, request).execute() }
```

Spaces within blocks `{ code }` is preferable for readability and distinguishing blocks from function calls.

Can you run code formatting on all your files?